src/stats.ts

```diff
@@ -179,6 +194,51 @@ export class StatsClient {
     return this.client.stats('redirect', params as Record<string, any>);
   }
 
+  /**
+   * Adds an amount of item to the cart in the current session. The cart will be automatically
+   * stored in stats if there's any call to registerCheckout. If the item is already in the
+   * cart, the amount is automatically added to the current amount.
```

Comment:
Use the same `@remarks` structure used [here](https://github.com/doofinder/webcomponents/pull/256).